test/unit/parser-test.js

```diff
@@ -69,13 +69,10 @@ describe('parser', function() {
         expect(dimension, 'dimension with empty string should be 0').to.equal(0);
 
         dimension = parser.parseDimension('35%');
-        expect(dimension, 'dimension with percentage string should be the same').to.equal('35%');
+        expect(dimension, 'dimension with percentage string should be the same').to.equal(35);
 
         dimension = parser.parseDimension('35px');
         expect(dimension, 'dimension with px string should remove px').to.equal(35);
-
-        dimension = parser.parseDimension(35);
-        expect(dimension, 'dimension with int should be itself').to.equal(35);
```

Comment:
I removed tests like this because they are technically invalid, and it felt weird keeping type checks inside the typed functions.

Reply:
Why is it invalid? Any of these inputs could be supplied as width or height in a player configuration.

Reply:
True. I hadn't thought about that. It just felt weird to have to keep type checks inside of typed functions.